packages/babel-preset/index.js

```diff
@@ -53,21 +55,27 @@ function getTargets(
   if (env !== 'production') {
     return target === 'node' ? { node: 'current' } : { esmodules: true };
   }
-  // TODO: distinguish between app and libraries for node as well
+
+  // TODO: Distinguish between app and libraries for node as well.
   if (target === 'node') return { node: '10.0' };
+
   if (
     ignoreBrowserslistConfig ||
-    !loadConfig({ path: configPath || path.resolve('.') })
-  )
-    return { browsers: targets.browsers || defaultBrowsers };
```

Comment:
`targets.browsers` is going away eventually. also `targets` might be null-y here